build/yamls/flow-aggregator.yml

```diff
@@ -6,6 +6,45 @@ metadata:
   name: flow-aggregator
 ---
 apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: flow-aggregator
+  name: flow-aggregator
+  namespace: flow-aggregator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: flow-aggregator
+  name: flow-aggregator-uuid-reader
```

Comment:
I'm waiting for https://github.com/vmware-tanzu/antrea/pull/1649 before I write the final version of this file, since there will be conflicts.

I doubt my PR will make it into v0.13.0 to be honest :/

Reply:
I think that's a good idea, even if only the flow aggregator needs that role for now. Pushing a commit with this change now.